From 4cc3cf84189c5fafc527469816be552b0f90fa05 Mon Sep 17 00:00:00 2001
From: Shivansh Rai <shivanshrai84@gmail.com>
Date: Sat, 16 Jul 2016 01:44:24 +0530
Subject: [PATCH] Add more options for tcp_info

---
 gtests/net/packetdrill/code.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gtests/net/packetdrill/code.c b/gtests/net/packetdrill/code.c
index e04a9ec..075012f 100644
--- a/gtests/net/packetdrill/code.c
+++ b/gtests/net/packetdrill/code.c
@@ -197,6 +197,10 @@ static void write_tcp_info(struct code_state *code,
 	emit_var(code, "tcpi_snd_ssthresh",	info->tcpi_snd_ssthresh);
 	emit_var(code, "tcpi_snd_cwnd",		info->tcpi_snd_cwnd);
 	emit_var(code, "tcpi_rcv_space",	info->tcpi_rcv_space);
+	emit_var(code, "tcpi_sacked",     info->__tcpi_sacked);
+	emit_var(code, "tcpi_unacked",    info->__tcpi_unacked);
+	emit_var(code, "tcpi_retransmits",    info->__tcpi_retransmits);
+	emit_var(code, "tcpi_retrans",    info->__tcpi_retrans);
 
 	/* FreeBSD extensions to tcp_info. */
 	emit_var(code, "tcpi_snd_wnd",		info->tcpi_snd_wnd);
-- 
2.7.2

